store/tikv/probe.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2021 PingCAP, Inc.
```

Comment:
e, could we rename it like test_probe.go or something similar

Reply:
yes